components/datepicker/utils.js

```diff
@@ -10,3 +12,35 @@ export const convertModifiersToClassnames = (modifiers, theme) => {
     };
   }, {});
 };
+
+export const hasRange = selectedDays => {
+  let containsFrom = false;
+  let containsTo = false;
+
+  if (!selectedDays || typeof selectedDays !== 'object') {
+    return false;
+  }
+
+  Object.keys(selectedDays).map(key => {
```

Comment:
Might want to use `Object.entries` here if you need both keys _and_ value?

Reply:
I removed this util function since we're nog using it anymore.

 :ok: by @driesd on 04/06/2018 12:20:33